scripts/pi-hole/js/db_graph.js

```diff
@@ -15,12 +15,6 @@ var interval = 0;
 
 var dateformat = "MMMM Do YYYY, HH:mm";
 
-// get the database min timestamp
```

Comment:
Why this was removed?

Are you reverting #1969 and #1976 ?

Reply:
Git accident. Multiple git directories with different state of `devel`. Going to fix this..